### PR TITLE
Test plugin for servus

### PR DESCRIPTION
**Pull request by eile**

### CODE REVIEW

**Comment by tribal-tec** [comment]
Changelog?

**Comment by eile** [comment]
Re Changelog: Is this a user feature, aka should we document it?

File: servus/test/servus.h [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+                     const unsigned browseTime) final
+    {
+        const servus::Servus::Result& result = beginBrowsing(addr);
+        if (!result && result != servus::Servus::Result::PENDING)
+            return getInstances();
+
+        browse(browseTime);
+        if (result != servus::Servus::Result::PENDING)
+            endBrowsing();
+        return getInstances();
```
[the remark this answers is not in the document]
Comment (eile) [reply servus/test/servus.h]: Even simpler. See a18cffe

[the remark this answers is not in the document]
Comment (rdumusc) [reply servus/test/servus.h]: yes, +1 for eliminating the duplication, this felt like it should be a function of the parent class.

**Comment by rdumusc** [comment]
I would also add a changelog entry, it doesn't cost anything and this is a new feature if you want to write unit tests like in ZeroEQ ;-)

**Comment by tribal-tec** [comment]
Definitely a user feature, as the user is a developer who uses Servus as library and he/she wants to write  unit tests using servus.

**Comment by rdumusc** [comment]
@eile the changelog is still missing... ;-)

**Pull request merged**